docs/src/index.md

```diff
@@ -181,8 +185,20 @@ linear (generic function with 2 methods)
 julia> grads = gradient(() -> sum(linear(x)), Params([W, b]))
 Grads(...)
 
+# Apply gradients to model parameters
 julia> grads[W], grads[b]
 ([0.652543 … 0.683588], [1.0, 1.0])
 ```
+Unlike with explicit gradients, in order to see implicit gradients one needs to do:
```

Comment:
We can index `Grads` directly. Why would you say one needs to access an internal field? That may break in future versions.